### PR TITLE
Web updates

### PR DESCRIPTION
**Pull request by Kryan92**

### CODE REVIEW

File: pages/what-we-do/partnerships/communities.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Digital communities of practice
+layout: basic
+permalink: /what-we-do/partnerships/communities/
```
Comment (bjgordon) [inline pages/what-we-do/partnerships/communities.md]: Add redirects from the old page:
/what-we-do/communities/
/our-work/communities/

File: pages/what-we-do/exemplar-projects.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Exemplar projects
+layout: basic
+permalink: /what-we-do/partnerships/exemplar-projects/
+localnav: what-we-do
+gov-speak: true
```
Comment (bjgordon) [inline pages/what-we-do/exemplar-projects.md]: We usually only use gov-speak for annual-reports and other long form documents. It's documented here: http://guides.service.gov.au/design-guide/foundations/typography/index.html#headings-guidance-accordion

File: pages/what-we-do/exemplar-projects.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Exemplar projects
+layout: basic
+permalink: /what-we-do/partnerships/exemplar-projects/
```
Comment (bjgordon) [inline pages/what-we-do/exemplar-projects.md]: add redirect from for /what-we-do/exemplars/

File: pages/what-we-do/partnerships/partnerships.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+permalink: /what-we-do/partnerships/
+localnav: what-we-do
+gov-speak: true
+---
+
+Building and delivering excellent services online requires a transformation of the way services are designed and delivered by government.
+
+The DTA works in close partnership with other government agencies to challenge and change the way services are traditionally delivered, and make them better.
+
+These partnerships range in scale and duration—from providing short-term specialist skills, support and training to long-term intensive partnerships from which major new online services are developed and delivered.
```
Comment (bjgordon) [inline pages/what-we-do/partnerships/partnerships.md]: Content guide says to use spaced em-dashes: http://content-guide.apps.staging.digital.gov.au/punctuation-grammar/#em-dashes-mdash

File: pages/who-we-are/who-we-are.md [hunk 5]
```diff
@@ -8,19 +8,17 @@ redirect_from:
...
-
-- to co-ordinate the funding of whole-of-government and shared ICT and digital service delivery platforms;
-
-- to manage a whole-of-government ICT program management office which will oversee all significant ICT and digital investments; and
-
-- to provide advice to the Minister on whole-of-government and shared ICT digital service delivery proposals.
+- user-centred research and design
+- agile delivery methodologies
+- technology design and build 
+- systems architecture, and
```
Comment (bjgordon) [inline pages/who-we-are/who-we-are.md]: The content guide recommends removing the ", and" - http://content-guide.apps.staging.digital.gov.au/content-structure/#bullet-point-lists

File: pages/what-we-do/exemplar-projects.md (hunk 2 above)
Comment (Kryan92) [inline pages/what-we-do/exemplar-projects.md]: Done

File: pages/what-we-do/partnerships/communities.md (hunk 1 above)
Comment (bjgordon) [inline pages/what-we-do/partnerships/communities.md]: Done

File: pages/what-we-do/exemplar-projects.md (hunk 3 above)
Comment (Kryan92) [inline pages/what-we-do/exemplar-projects.md]: Old page still exists, is linked to in new page

File: pages/what-we-do/partnerships/partnerships.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+title: Partnerships
+layout: basic
+permalink: /what-we-do/partnerships/
+localnav: what-we-do
+gov-speak: true
```
Comment (bjgordon) [inline pages/what-we-do/partnerships/partnerships.md]: Remove govspeak

File: pages/what-we-do/partnerships/partnerships.md (hunk 4 above)
Comment (Kryan92) [inline pages/what-we-do/partnerships/partnerships.md]: Done

File: pages/what-we-do/partnerships/communities.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+localnav: what-we-do
+gov-speak: true
```
Comment (bjgordon) [inline pages/what-we-do/partnerships/communities.md]: I've removed this gov-speak

File: pages/who-we-are/who-we-are.md (hunk 5 above)
Comment (Kryan92) [inline pages/who-we-are/who-we-are.md]: Done

File: pages/what-we-do/partnerships/partnerships.md (hunk 6 above)
Comment (Kryan92) [inline pages/what-we-do/partnerships/partnerships.md]: Done

**Pull request merged**